src/shared/install.h

```diff
@@ -78,6 +79,13 @@ enum UnitFileChangeType {
         _UNIT_FILE_CHANGE_INVALID = INT_MIN
 };
 
+enum UnitFileFlags {
+        UNIT_FILE_RUNTIME = 1,
+        UNIT_FILE_FORCE = 1 << 1,
+        _UNIT_FILE_FLAGS_MAX,
```

Comment:
i am not sure there's much value in defining _UNIT_FIULE_FLAGS_MAX here... after all it, if you add one more flag (which would then be 1 << 2, i.e. 4), it would end up being 5, which is not useful at all, in no way possible...